beakerx/beakerx/static/html/settings_tab.html

```diff
@@ -15,15 +15,33 @@
 -->
 <div id="beakerx" class="tab-pane">
     <style>
-        .bko-spacing .form-control { margin-right: 10px; }
-        #beakerx_info .beakerx_site_link { display: inline-block; margin: 5px 0; }
-        #beakerx_info .beakerx_site_link img { height: 35px; }
+        .bko-spacing .form-control {
+            margin-right: 10px;
+        }
+
+        #beakerx_info .beakerx_site_link {
+            display: inline-block;
+            margin: 5px 0;
+        }
+
+        #beakerx_info .beakerx_site_link img {
+            height: 35px;
+        }
     </style>
     <div id="beakerx_env_toolbar" class="list_toolbar row">
-        <div class="col-xs-12 no-padding">
-           <span id="beakerx_info" class="toolbar_info"></span>
+        <div class="col-xs-11 no-padding">
+            <span id="beakerx_info" class="toolbar_info"></span>
         </div>
-    </div><br/>
+        <div class="col-xs-1">
+           <span id="beakerx_java" class="toolbar_info">
+
+            <a class="beakerx_javadoc_link" href="beakerx/javadoc/index.html" target="_blank">
+            <input type="button" class="button" value="Java Doc"/>
```

Comment:
instead of putting a button in the settings tab, add an item to the Help menu, like https://github.com/twosigma/beakerx/blob/ffd6f5cdf764814ce1bba815cf407b2dfcce261b/kernel/base/src/main/java/com/twosigma/beakerx/KernelInfoHandler.java#L73

Reply:
Done